### ISSUE TITLE
How to use TimeDistributedMerge using TimeDistributed wrapper ?

### ISSUE THREAD

**Issue by vedavyas777** [fchollet/keras#2683]
I am using TimeDistributedMerge to merge outputs from lstm over time by returning sequences, since TimeDistributedMerge is deprecated, so I changed my implementation to use timedistributed wrapper then it gives error.

File "/Library/Python/2.7/site-packages/keras/engine/topology.py", line 148, in create_node
    output_tensors = to_list(outbound_layer.call(input_tensors[0], mask=input_masks[0]))
  File "/Library/Python/2.7/site-packages/keras/layers/wrappers.py", line 131, in call
    y = self.layer.call(X)  # (nb_samples \* timesteps, ...)
  File "/Library/Python/2.7/site-packages/keras/engine/topology.py", line 1189, in call
    '(at least 2). Got: ' + str(inputs))
Exception: Merge must be called on a list of tensors (at least 2). Got: Reshape{2}.0

My code
model = Sequential()
model.add(Embedding(max_features, 128, input_length=chunk_size, dropout=0.5))
model.add(LSTM(128, dropout_W=0.5, dropout_U=0.1,return_sequences=True))
model.add(TimeDistributed(Merge(layers=None,mode='ave')))
model.add(Dense(num_classes))
model.add(Activation("softmax"))

@fchollet can you help me out with this by giving an example of how to use timedistributed wrapper for lstm return sequences

**Comment by akansal1** [comment]
you can use:
```python

model.add(TimeDistributed(Dense(output_dim= , activation= ))))
```
to pass your LSTM layer output to time distributed layer

**Comment by carlthome** [comment]
Duplicate. Please search: https://github.com/fchollet/keras/issues/2271

**Comment by ymcui** [comment]
write a lambda layer by yourself
for e.g.
```
def td_sum(x):
    return K.sum(x,axis=1)
```
